### ISSUE TITLE
Small text improvement 

### ISSUE THREAD

**Issue by Davi427** [osmandapp/OsmAnd#21387]
### Describe the idea

This text could be clearer that only the one profile will be reset. I was worried that all profile settings would be reset, since that is what the text says.

![Screenshot_20241119-084437](https://github.com/user-attachments/assets/ac773142-4cbe-40d4-9362-e244ceb4f557)

### Expected behaviour

Reset [profile name] settings?
and
Reset [profile name] settings to installation defaults.

### Alternatives you've considered

I have not considered alternatives.

**Comment by yuriiurshuliak** [comment]
I’d like to clarify how this feature works:

If you choose to reset settings within a specific profile, the changes will only apply to that profile.
In other cases (e.g., global settings), a context menu will appear, allowing you to select whether to apply the reset to the current profile only or to all profiles.

<table style="width:100%">
  <tr>
<th><img src="https://github.com/user-attachments/assets/73a3a204-20d1-4be8-a85a-a7c168f4a0bf"  width="300" height="auto"/> </th>
  </tr>
</table>

**Comment by vshcherb** [comment]
Yes, it states "Reset profile settings", could be phrased better somehow "Reset all only this profile settings" but I couldn't decide how to phrase it correctly

**Comment by MakarenkoAnastasia** [comment]
It is unclear which profile settings will be reset due to an incorrect setting name.

![IMAGE 2024-11-24 13:05:18](https://github.com/user-attachments/assets/bccd82d2-f342-4008-87bd-50fc454c40bc)
OsmAnd Version: 5.0.0#3792m
Android version: android 13
Device model: Pixel 6a
Crash-Logs: ---

**Comment by yuriiurshuliak** [comment]
I can confirm that on my end, the moped is also displayed as a cycling during a profile reset. As I understand it, this is because the moped and cycling share the same routing parameters and behave similarly in terms of road usage.

**Comment by scaidermern** [comment]
> but I couldn't decide how to phrase it correctly

The original author already suggested "Reset [profile name] settings". This is concise and easy to understand.
